src/partials/50_section-products.html

```diff
@@ -176,12 +180,15 @@ <h5>Aenean Ru Bristique</h5>
               <a class="star black-outline"></a>
               <a class="star black-outline"></a>
             </div>
-          </div>
-          <div class="line"></div>
-          <div class="actions">
-            <div class="outlines">
-              <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
-              <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+            <div class="line"></div>
+            <div class="actions">
+              <div class="outlines">
+                <a href="#" class="btn-outline"><i class="far fa-heart"></i></a>
+                <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
+              </div>
+              <div class="price">
+                <div class="btn-main-small no-hover">$ 30.00</div>
+              </div>
             </div>
             <div class="price"><div class="btn-main-small no-hover">$ 15.00</div></div>
           </div>
```

Comment:
W tym przypadku chcąc poprawić formatowanie div'a o klasie price zapomniałaś usunąć poprzedniego div'a i elementy się rozepchały, w poniższym kodzie jest tak samo

Reply:
Poprawiłam to o czym pisaleś (price, zakończenie diva my-slider) i dalej źle się wyświetla... Nie mam już pomysłu :(